### DISCUSSION TITLE
Ban Appeal

### DISCUSSION THREAD

**Discussion by cnkelvin** [MenheraBot/MenheraBot#224] [Ban Appeal]
Qual o seu ID do Discord?

`532694099674005547`

Quando foi o seu banimento?

`08 de março de 2022 às 21:10`

Por que tu foi banido?

`[AUTOBAN] - Divulgação de redes sem permissão`

Coloque o motivo de seu banimento

`[AUTOBAN] - Divulgação de redes sem permissão`

Por que tu fez isso?

`Sinceramente, eu não lembro de ter divulgado nenhum link no server da Menhera. Eu raramente conversei e o que eu conversei foi sobre a construção do RPG e afins.`

Qual regra que tu quebrou fazendo isso?

`Divulgação de redes sem permissão`

Uma mensagem para a Lux

`Se realmente aconteceu de eu enviar um link indevido, peço desculpas, não queria causar transtornos no server. Eu fui conferir o meu perfil agora (por causa do anúncio dos 3 anos da Menhera) e vi só hoje essa mensagem de banimento. Enfim, eu queria muito saber qual foi esse link que eu divulguei pois não lembro dele kk. Eu sei que é meio tarde fazer esse ban appeal quase 1 ano depois do banimento, mas eu não sabia que fui banido e eu estava usando bastante o bot por causa do hype do RPG na época. Decidi voltar agora pra ver como estava e é isso. Feliz 3 anos da Menhera, espero que você continue desenvolvendo o seu bot e acreditando no seu potencial, continue com esse carinho e criatividade. Eu espero que você alcance tudo o que você deseja conquistar. Abraços!`

Você concorda com os Termos de Uso e Privacidade da Menhera?

[X] Sim, eu concordo.

**Comment by ySnoopyDogy** [comment]
Eu também não sei qual foi a mensagem do Ban, já que foi um ban automático por compartilhar links.

Eu tirei teu ban, e tu ja pode voltar a usar a Menhera :heart:
